packages/shared/src/components/PostOptionsReadingHistoryMenu.tsx

```diff
@@ -88,6 +88,7 @@ export default function PostOptionsReadingHistoryMenu({
   const { user } = useContext(AuthContext);
   const queryClient = useQueryClient();
   const historyQueryKey = ['readHistory', user?.id];
+  const hasEngagementLoopAccess = true;
```

Comment:
Just blocking note this needs to be experiment, can't we overwrite it locally instead?